Sources/Valet/ErrorHandler.swift

```diff
@@ -1,49 +0,0 @@
-//
-//  ErrorHandler.swift
-//  Valet
-//
-//  Created by Dan Federman and Eric Muller on 9/16/17.
-//  Copyright © 2017 Square, Inc.
-//
-//  Licensed under the Apache License, Version 2.0 (the "License");
-//  you may not use this file except in compliance with the License.
-//  You may obtain a copy of the License at
-//
-//    http://www.apache.org/licenses/LICENSE-2.0
-//
-//  Unless required by applicable law or agreed to in writing, software
-//  distributed under the License is distributed on an "AS IS" BASIS,
-//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-//  See the License for the specific language governing permissions and
-//  limitations under the License.
-//
-
-import Foundation
-
-
-public final class ErrorHandler {
```

Comment:
We no longer needed a hot-swappable `ErrorHandler`. We now `throw` when an error occurs. We also have `assertionFailure` in a few `internal` methods where the error would very much be a programmer error _within_ Valet.